packages/dev/package-vite-config/package.json

```diff
@@ -31,8 +31,7 @@
   },
   "dependencies": {
     "vite": "5.3.4",
-    "vite-plugin-dts": "3.9.1",
-    "vitest": "^2.0.3"
```

Comment:
I think vitest is not needed here

Reply:
it's referenced in the config itself, see:
```ts
import { type UserConfig, defineConfig } from 'vitest/config'
```

Reply:
Good catch! sorry I missed it 😓 added it back